apps/www/app/code-examples.tsx

```diff
@@ -472,7 +338,7 @@ public class APIController {
 
 type Framework = {
   name: string;
-  Icon: React.FC<IconProps>;
+  Icon: React.FC<LangIconProps>;
```

Comment:
It makes sense to change them to `tsx` because `ts`, `java` and `elixir` is [not supported out-of-box by prism-react-renderer](https://github.com/FormidableLabs/prism-react-renderer/blob/c914fdea48625ba59c8022174bb3df1ed802ce4d/packages/generate-prism-languages/index.ts#L9-L23).